radicle-node/src/service.rs

```diff
@@ -542,9 +543,8 @@ where
     pub fn fetched(&mut self, fetch: Fetch, result: Result<Vec<RefUpdate>, FetchError>) {
         let remote = fetch.remote;
         let rid = fetch.rid;
-        let initiated = fetch.initiated;
 
-        if initiated {
+        if let Some(namespaces) = fetch.namespaces() {
```

Comment:
I feel like this is a bit obscure, I would rather something like `if let FetchDirection::Initiated { namespaces } = fetch {`, otherwise it's not clear what's significant about the namespaces being optional.

Reply:
Ah, that's a fair point. I actually had that originally but then opted for introducing this method. Just as another suggestion, what do you think of renaming the method:
```rust
pub fn receiving(&self) -> Option<&Namespaces> {
/* etc. */
}
```

Otherwise, happy to change it to the `if let` :+1:

Reply:
Yeah, see my other comments about "receiving"; but I think something like `if let Some(ns) = fetch.initiated() {` could work.